### ISSUE TITLE
Is the HTML output published as a web page anywhere?

### ISSUE THREAD

**Issue by mlinksva** [finos-osr/OSLC-handbook#14]
It'd be helpful to be able to link to e.g., `SOME_STABLE_LOCATION#_gnu_lesser_general_public_license_2_1` and expect link not to break, rather than linking to a page to download the handbook and telling recipient to look up what it says on some topic.

**Comment by copiesofcopies** [comment]
Good point, @mlinksva. It's not yet, mostly because we ran out of time to make it part of the initial release, but we'll be putting one up ASAP.

**Comment by mlinksva** [comment]
I don't know why I didn't say when filing, it'd be even more helpful if whatever link to specific license info is eventually published if the identifier used were the license's SPDX ID so that the particularly name mangling used in the handbook doesn't have to be looked up for each license linked to (or be expected to change).
